contracts/teller-network/Escrow.sol

```diff
@@ -185,4 +278,119 @@ contract Escrow is Pausable {
         trx.rating  = _rate;
         emit Rating(trx.seller, trx.buyer, _escrowId, _rate);
     }
+
+    modifier onlyArbitrer {
```

Comment:
Is it normal there is no test for all the function below?

Reply:
I forgot to add tests for `setArbitrer()` and `isArbitrer()`.  Good catch!